10up/maps-block-apple
# Add PHP version check for plugin.

### Description of the Change

Add minimum version check for the plugin before loading it. This will ensure plugin update doesn't break the site that don't match our minimum PHP version.

<img width="1039" alt="image" src="https://github.com/10up/maps-block-apple/assets/10358350/cea3e319-19e5-47ab-a221-35b2398262dc">



Closes #172

### How to test the Change


1. Setup a WordPress environment running PHP 7.3
2. Install Block for Apple Maps plugin by downloading it with this [feature](https://github.com/10up/maps-block-apple/archive/refs/heads/fix/172-php-ver-check.zip) branch.
3. Verify that plugin functionality doesn't load and you see an admin error message for php version. ( refer above screenshot in Description )

### Changelog Entry

> Fixed - Better error handling for environments that don't match our minimum PHP version

### Credits

Props @dkotter, @rahulsprajapati 


### Checklist:
